lib/utils/reader.cpp

```diff
@@ -106,62 +128,67 @@ void Reader::readProblem(const std::string problems_filepath, const core::int_t
 
   // get number of variables and number of inequalities we are reading in
   std::getline(newfile, num_variables_string);
-  const core::int_t num_variables = static_cast<core::int_t>(atoi(num_variables_string.c_str())) + 1;
+  const int num_variables = static_cast<int>(atoi(num_variables_string.c_str()));
```

Comment:
Since we changed this to base `int`, we don't need to static cast any more because `atoi` returns an integer anyways.